Dockerfile

```diff
@@ -1,4 +1,13 @@
 FROM google/dart:2.13
+
+WORKDIR /build
 COPY pubspec.yaml .
+COPY /lib ./lib/
+
 RUN dart pub get
+RUN cd lib && tar -czvf ../cdn_assets.tar.gz *.js
+
+ARG BUILD_ARTIFACTS_AUDIT=/build/pubspec.lock
```

Comment:
I don't think you need to explicitly provide this audit artifact. Workiva build will inspect each layer of your docker image and find this file on its own.